package.json

```diff
@@ -121,7 +121,7 @@
     "cover": "istanbul cover test/index.js",
     "lint": "standard && tslint -c tslint.json --project .",
     "sinopia": "sinopia -c ./sinopia/config.yaml",
-    "test:tap": "ts-node test | tap-diff",
+    "test:tap": "ts-node test | tap-pessimist",
```

Comment:
well, it should show only the part of the raw tap output that have failed. But I checked and it strips out too much, doesn't show the expected/actual text and the row where the error happened. So I'll try some other alternative

I want this because there are too many tap log and when something fails it is very hard to find the error message. Especially in AppVeyor